tests/test_github_reviews.py

```
@@ -0,0 +1,102 @@
+        {"id": 2, "body": "<!-- cerberus:council-review sha=abc -->\nhi"},
+    ]
+    assert (
+        find_review_id_by_marker(reviews, "<!-- cerberus:council-review sha=abc -->")
+        == 2
+    )
+
+
+def test_list_pr_reviews_parses_json_list(monkeypatch) -> None:
+    calls: list[list[str]] = []
+
+    def mock_run_gh(args, *, check=True, max_retries=3, base_delay=1.0):
```

Ruff (0.15.0)

[warning] Unused function argument: `check`

(ARG001)